src/Service/LabelNameExtractor.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Service;
+
+use App\Issues\GitHub\CachedLabelsApi;
+use App\Repository\Repository;
+
+/**
+ * Extract label name from a PR/Issue.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class LabelNameExtractor
+{
+    private $labelsApi;
+
+    private static $labelAliases = [
+        'di' => 'DependencyInjection',
+        'bridge\twig' => 'TwigBridge',
+        'router' => 'Routing',
+        'translation' => 'Translator',
+        'twig bridge' => 'TwigBridge',
```

Comment:
is this frequent enough?
for bridges, all of them can be FooBridge or Bridge[\/]Foo, maybe we should have some logic for them?

Reply:
Im not sure..
This list is super old. I just moved it. 

I'll add other bridges too